themes/docs-new/layouts/partials/sidebar-on-canvas.html

```diff
@@ -6,7 +6,7 @@
     {{ if .Site.Params.enable_search }}
       <div class="search">
         <form>
-          <input type="text" class="st-default-search-input" placeholder="Search...">
+          <input type="text" class="swiftype-search-input swiftype-search-autocomplete" placeholder="Search...">
         </form>
```

Comment:
for accessibility, All inputs need a label associated - which can be hidden for this type of styling
i.e.
```
<form role="search" aria-label="documentation">
  <label class="some-class-that-hides-me" for="search-input">Search the docs</label>
  <input type="search" id="search-input">
</form>
```

See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Search_role

Reply:
According to this we can use `aria-label` on the input tag and then I don't have to hide something. 
https://www.w3.org/WAI/tutorials/forms/labels/#using-aria-label

Reply:
Beautiful!  good find!